README.md

```diff
@@ -4,3 +4,199 @@ Unikernel Alliance gathers all unikernel and library OS projects under one umbre
+
+```
+
+### (b) Build static site:
+
+```
+npm run build
+```
+
+The static website should be in the `public` directory:
+
+```
+$ tree public/
+public/
+├── 404.html
+├── apple-touch-icon.png
+├── blog
+│   ├── example-post
+│   │   └── index.html
+│   ├── index.html
+│   ├── index.xml
+│   ├── page
+│   │   └── 1
+│   │       └── index.html
+│   └── sitemap.xml
+├── categories
+│   ├── index.html
+│   ├── index.xml
+│   └── page
+│       └── 1
+│           └── index.html
+├── contributors
+│   ├── index.html
+│   ├── index.xml
+│   └── page
+│       └── 1
+│           └── index.html
+├── cover.png
+├── docs
+│   ├── guides
+│   │   ├── example-guide-111
+│   │   │   └── index.html
+│   │   ├── index.html
+│   │   ├── index.xml
+│   │   └── sitemap.xml
+│   ├── index.html
+│   ├── index.xml
+│   ├── reference
+│   │   ├── example-reference
+│   │   │   └── index.html
+│   │   ├── index.html
+│   │   ├── index.xml
+│   │   └── sitemap.xml
+│   └── sitemap.xml
+├── en
+│   ├── index.html
+│   └── sitemap.xml
+├── favicon-150x150.png
+├── favicon-16x16.png
+├── favicon-192x192.png
+├── favicon-32x32.png
+├── favicon-512x512.png
+├── favicon.ico
+├── favicon.svg
+├── fonts
+├── index.html
+├── index.xml
+├── js
+│   ├── app.1e84dcdaa5ccef6fe83f6d88ac1e444a1f237111cfa2a7a97427f758e9c27ef4.js
+│   ├── color-mode.86a91f050a481d0a3f0c72ac26543cb6228c770875981c58dcbc008fd3f875c8.js
+│   ├── flexsearch.78e4b7a9456a2770bd6faffb1f6488b30db12ca157c3df4e89e3ae2e258888ae.js
+│   └── search-modal.c4c023efcdee308e4b5f1110afe1db6b1656f217e67aa7efe54e2f4bf2feccf8.js
+├── main.77ec3b13291e19da56fd3d976a516a363623a782a2d5533bb4421e553b5207ee09816f4a448adac85cff292ac1ab9f46d86e64bd0f325c95beb792912a0a2bd1.css
+├── mask-icon.svg
+├── privacy
+│   └── index.html
+├── projects
+│   ├── bima
+│   │   └── index.html
+│   ├── flexos
+│   │   └── index.html
+│   ├── hermitux
+│   │   └── index.html
+│   ├── index.html
+│   ├── index.xml
+│   ├── loupe
+│   │   └── index.html
+│   ├── osv
+│   │   └── index.html
+│   ├── sitemap.xml
+│   ├── unikraft
+│   │   └── index.html
+│   └── urunc
+│       └── index.html
+├── robots.txt
+├── search-index.json
+├── sitemap.xml
+└── tags
+    ├── index.html
+    ├── index.xml
+    └── page
+        └── 1
+            └── index.html
+
+33 directories, 131 files
+```
+
+and the output something like the following:
+
+```
+$ npm run build
+
+> unikernelalliance.org@0.0.0 build
+> exec-bin node_modules/.bin/hugo/hugo --cleanDestinationDir --minify
+
+Start building sites … 
+hugo v0.119.0-b84644c008e0dc2c4b67bd69cccf87a41a03937e+extended linux/amd64 BuildDate=2023-09-24T15:20:17Z VendorInfo=gohugoio
+
+
+                   | EN  
+-------------------+-----
+  Pages            | 37  
+  Paginator pages  |  0  
+  Non-page files   |  0  
+  Static files     | 72  
+  Processed images | 16  
+  Aliases          |  5  
+  Sitemaps         |  2  
+  Cleaned          |  0  
+
+Total in 1150 ms
+```
+
+#### Auto-deploy on github pages:
```

Comment:
```suggestion
## Auto-deploy on GitHub Pages
```

Reply:
TODO: capitalize github pages, will do that on the 2nd iteration of pushes.